.github/workflows/docs.yml

```diff
@@ -3,7 +3,17 @@ name: Build docs
 on:
   push:
     branches: [master]
+    paths-ignore:
+      - "tests/**"
+      - "docker/**"
+      - "assets/**"
+      - ".github/workflows/**"
   pull_request:
+    paths-ignore:
+      - "tests/**"
+      - "docker/**"
+      - "assets/**"
```

Comment:
ok, `tests` and `docker` are fine, but we use some items from `assets`, so let's remove `assets`

```suggestion
```